drv/gimlet-seq-server/src/main.rs

```diff
@@ -307,35 +297,52 @@ fn main() -> ! {
         hl::sleep_for(1);
     }
 
+    //
+    // If our clock generator is configured to load from external EEPROM,
+    // we need to wait for up to 150 ms here (!).
+    //
+    hl::sleep_for(150);
+
+    //
+    // And now load our clock configuration
+    //
+    let mut packet = 0;
+    let clockgen = i2c_config::devices::idt8a34003(I2C.get_task_id())[0];
+
+    payload::idt8a3xxxx_payload(|buf| match clockgen.write(buf) {
+        Err(err) => Err(err),
```

Comment:
It looks like we previously logged these results in the ringbuf – is it deliberate to not log them now?

If so, this could be slightly simplified to
```rust
|buf| {
    clockgen.write(buf)?;
    packet += 1;
    Ok(());
}
```

Reply:
I *did* purposely not log them -- and agreed on the simplification!